src/Item.h

```diff
@@ -201,6 +201,16 @@ class cItem
 	Returns true if the item was enchanted, false if not (not enchantable / too many enchantments already). */
 	bool EnchantByXPLevels(int a_NumXPLevels);  // tolua_export
 
+	/** Adds this specific enchantment to this item, returning the cost */
+	int AddEnchantment(int a_EnchantmentID, unsigned int a_Level, bool a_WithBook);  // tolua_export
+
+	/** Adds the enchantments on a_Other to this item, returning the
+	XP cost of the transfer. */
+	int AddEnchantmentsFromItem(cItem & a_Other);  // tolua_export
```

Comment:
It does not, but internally it requires cItem's begin() and end() to return const_iterators, which I just implemented.